src/template.ts

```diff
@@ -243,7 +243,7 @@ template.innerHTML = `
 
     <div class="slot ar-button">
       <slot name="ar-button">
-        <a id="default-ar-button" class="fab" href="#"
+        <a id="default-ar-button" class="fab"
```

Comment:
What was href="#" for?

Reply:
The reason to put it on a valid `href` attribute on an `<a>` is because the browser gives anchors different _default_ styles depending on whether or not they have an `href` attribute, and `href="#"` is a commonly used value that won't cause a navigation. We no longer depend on the default styles for `<a>` and we also `preventDefault()` on the event, so the attribute is vestigial.